test/e2e/pages/move-detail.js

```diff
@@ -35,15 +35,15 @@ class MoveDetailPage extends Page {
       ),
       riskInformation: Selector('#main-content h2')
         .withText('Risk information')
-        .sibling('.app-panel'),
+        .parent('section'),
```

Comment:
Is this just a looser (and higher in the DOM) selector to catch both versions... or have we actually changed the markup?

Reply:
When this was originally written we didn't have the wrapping parent `section` I believe. This just changes to to encapsulate the whole container to make a few other checks, like checking for content, a bit easier.